scripts/globals/additional_effects.lua

```diff
@@ -285,6 +286,14 @@ xi.additionalEffect.attack = function(attacker, defender, baseAttackDamage, item
             msgParam = xi.effect.KO
             defender:setHP(0)
         end
+
+    elseif addType == procType.LOCAL_VAR then
```

Comment:
This looks like it's adding a new proc type, but it's entirely hard-coded for Pinch Of Stygian Ash, this is probably not a good pattern

Reply:
it's hardcoded to apply this particular LocalVar for stygian ash but easily extensible if we had the need for some other AE that applied a localvar on the target.